app/migrations/fromGE/rds_migrate.py

```diff
@@ -0,0 +1,315 @@
+                                          db=SECRETS.name,
+                                          user=SECRETS.user,
+                                          port=SECRETS.port,
+                                          password=SECRETS.password,
+                                          connect_timeout=timeout)
+
+        if not self.connection:
+            raise RuntimeError('Could not connect to database')
+        else:
+            print('DB connection successfully created.  Yeah us.')
+
+    def migrate(self):
+        self.migrateUsers()
+        self.migratePOA()
+
+
+    def migrateUsers(self):
+        list_of_ge_ids = self.get_ids_from_sempo()
+        print('list_of_ge_ids', list_of_ge_ids)
+        self.get_new_users_from_GE(list_of_ge_ids)
+        self.update_users_refered_by()
+
+    def get_ids_from_sempo(self):
+        sql = '''SELECT
+            c1.value::text::int
+            FROM "user" u1
+            INNER JOIN custom_attribute_user_storage c1 ON u1.id = c1.user_id
+            WHERE c1."name" = 'GE_id'
+        '''
+        result = db.session.execute(sql)
+        list_of_ids = [r[0] for r in result]
+        if len(list_of_ids) > 0:
+            return list_of_ids
+        else:
+            return [0]
+
+
+    def get_new_users_from_GE(self, list_of_ge_ids):
+        print('Getting new users from GE')
+        with self.connection.cursor(MySQLdb.cursors.DictCursor) as cursor:
+            format_strings = ','.join(['%s'] * len(list_of_ge_ids))
+            cmd = """SELECT * FROM users
+                LEFT JOIN community_tokens ON users.community_token_id=community_tokens.id
+                LEFT JOIN business_categories ON users.business_category_id=business_categories.id
+                LEFT JOIN token_agents on users.id=token_agents.user_id
+                LEFT JOIN group_accounts on users.id=group_accounts.user_id
+                WHERE users.id NOT IN (%s) LIMIT 100
+            """
+            cursor.execute(cmd % format_strings, tuple(list_of_ge_ids))
+            users = cursor.fetchall()
+            i = 0
+            print('Fetched {} users'.format(len(users)))
+
+            n_users = len(users)
+            t0 = time.time()
+            estimate_time_left = 'unknown'
+            for user in users:
+                i += 1               
+                print('Adding user {} of {}. User name = {} {}. Estimated time left {}. seconds'.format(
+                        i, n_users, user['first_name'], user['name'], estimate_time_left))
+                pprint.pprint(user)
+                self.insert_user(user)
+                elapsed_time = time.time() - t0
+                estimate_time_left = round((elapsed_time / i * n_users) - elapsed_time, 1)
+
+    def insert_user(self, ge_user):
+        if ge_user['business_type'] is not None:
+            transfer_usage = TransferUsage.find_or_create(ge_user['business_type'])
+            business_usage_id = transfer_usage.id
```

Comment:
this will fail if the transfer usage doesn't exist yet: TODO - swap to passing transient object